docs/concepts/validators.md

```diff
@@ -802,6 +802,33 @@ Pydantic provides a few special utilities that can be used to customize validati
     1. Note that the validation of the second item is skipped. If it has the wrong type it will emit a
        warning during serialization.
 
+* [`ValidateFrom`][pydantic.functional_validators.ValidateFrom] can be used to validate an custom type from a
```

Comment:
`ValidateFrom` is a bit unclear to me, what do you think about `ValidateAs` or `ValidateWith`?